docs/aws_storage.md

```diff
@@ -0,0 +1,41 @@
+# AWS Storage setup instruction for Loki
```

Comment:
I'd probably prefer if this was rolled into the operations.md file.

Reply:
Ok, I will roll this into operations.md.  I was a little afraid of the operations readme become too long, that's why I split out the readme.